components/list/test/list.vdiff.js

```diff
@@ -419,6 +420,52 @@ describe('list', () => {
 		});
 	});
 
+	[true, false].forEach(disabled => {
+		describe(`nav${disabled ? '-disabled' : ''}`, () => {
+			[
+				{ name: 'default' },
+				{ name: 'focus', action: focusElem, margin: disabled ? undefined : 24 },
+				{ name: 'focus add-button', action: focusElem, margin: disabled ? 75 : 24, addButton: true },
+				{ name: 'hover', action: hoverElem, margin: disabled ? undefined : 24 }
+			].forEach(({ name, action, margin, addButton }) => {
+				it(name, async() => {
+					const elem = await fixture(html`
+						<d2l-list style="width: 400px;" ?add-button="${addButton || false}">
+							<d2l-list-item-nav ?button-disabled="${disabled}">
+								${interactiveListItemContent}
+							</d2l-list-item-nav>
+						</d2l-list>
+					`);
+					if (action) await action(elem.querySelector('d2l-list-item-nav'));
+					await expect(elem).to.be.golden({ margin });
+				});
+
+				it(`nested-${name}`, async() => {
+					if (addButton) return;
```

Comment:
There's a bug here. It's not a problem with this implementation specifically, but with `add-button` in a nested list and `draggable`. I'll be opening up a separate defect.
<img width="1177" alt="Screenshot 2025-04-22 at 10 05 48 AM" src="https://github.com/user-attachments/assets/66fbd535-b564-4572-b8c5-0ad7f0d26af2" />